src/pages/project/project_wizard/ProjectFormStepper.tsx

```diff
@@ -34,7 +34,7 @@ export const ProjectFormStepper = ({
     }
   };
 
-  return (
+  return isMobile ? null : (
     <Stepper nonLinear activeStep={tabNumber}>
```

Comment:
Kunne også vært løst med ren CSS, sånn at det ikke vil trigge noen re-render. Ulempen(?) er at HTMLen fortsatt vil ligge i treet da, den bare skjules i CSS. Vet ikke hva som egentlig er beste practice av dem 
```suggestion
    <Stepper nonLinear activeStep={tabNumber} sx={{ display: { xs: 'none', md: 'flex' } }}>
```

Reply:
Ja vet heller ikke. Dette er sånn det er gjort i den andre stepperen. Tror ikke det er noe problem at den ligger skjult i treet, men da blir den også rendret da.